tasks/clean_pack.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+# Start in tests/ even if run from root directory
+cd "$(dirname "$0")"
+
+# Exit the script on any command with non 0 return code
+# We assume that all the commands in the pipeline set their return code
+# properly and that we do not need to validate that the output is correct
+set -e
```

Comment:
Can you please use the same error handling mechanism that e2e test [now uses](https://github.com/facebookincubator/create-react-app/blob/0e5abada75db99d3f8d3598afe013c21dc810e85/tasks/e2e.sh#L11-L35)?